examples/generative/md/text_generation_with_miniature_gpt.md

```diff
@@ -326,236 +349,1261 @@ model.fit(text_ds, verbose=2, epochs=25, callbacks=[text_gen_callback])
 <div class="k-default-codeblock">
 ```
 Epoch 1/25
-391/391 - 135s - loss: 5.5949 - dense_2_loss: 5.5949
+
+WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
```

Comment:
I recommend hand-scrubbing all logs like this because they don't add value in the rendered page.